clang-tools-extra/docs/clang-tidy/checks/readability/enum-initial-value.rst

```diff
@@ -0,0 +1,46 @@
+.. title:: clang-tidy - readability-enum-initial-value
+
+readability-enum-initial-value
+==============================
+
+Detects explicit initialization of a part of enumerators in an enumeration, and
+relying on compiler to initialize the others.
+
+It causes readability issues and reduces the maintainability. When adding new
```

Comment:
no need for " readability issues and reduces the maintainability", just say that this is about consistency.